### PR TITLE
Add pass to convert split to many slice

### PR DESCRIPTION
**Pull request by Erik-Lundell**
Change-Id: I15f8a37fe88ba6c7d1e357c9ab897069c3c912b2

### CODE REVIEW

File: backends/arm/passes/convert_split_to_slice.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+# LICENSE file in the root directory of this source tree.
+
+import torch.fx
+from executorch.backends.arm.tosa_mapping import extract_tensor_meta
+from executorch.exir.dialects._ops import ops as exir_ops
+from executorch.exir.pass_base import ExportPass, PassResult
+from torch.fx.passes.utils.source_matcher_utils import get_source_partitions
+
+
+class ConvertSplitToSlicePass(ExportPass):
```
Comment (digantdesai) [inline backends/arm/passes/convert_split_to_slice.py:14]: Nice, like it how simple you made it. :)

**Review by digantdesai — APPROVED** [review]

**Comment by digantdesai** [comment]
Seems like I messed up the merge conflict resolution. Let me fix the missing `)`.

**Comment by digantdesai** [comment]
Actually can you please rebase (force) since there are more conflicts. Thanks.

**Comment by Erik-Lundell** [comment]
> Actually can you please rebase (force) since there are more conflicts. Thanks.

Done :)

**Pull request merged**